---
course_id: 7-342-systems-and-synthetic-biology-how-the-cell-solves-problems-fall-2010
layout: course_section
menu:
  leftnav:
    identifier: 35ac297c8a548dc91ccf75219daf7636
    name: Lecture Summaries
    weight: 20
title: Lecture Summaries
type: course
uid: 35ac297c8a548dc91ccf75219daf7636

---

| WEEK # | TOPICS | LECTURE SUMMARIES |
| --- | --- | --- |
| 1 | Introduction to the class and topic | {{< anchor "The_instructor_and_students" >}}{{< /anchor >}}The instructor and students will introduce themselves. The instructor will discuss the administrative aspects of the course and present a general overview of systems and synthetic biology and in particular the aspects that will be the focus of the course. This summary is the only lecture that will be given in the course, and we hope for an interactive environment even during this first day. |
| 2 | Simple synthetic networks | {{< anchor "Much_of_the_recent" >}}{{< /anchor >}}Much of the recent interest in systems biology has been motivated by the need to understand how simple artificially constructed genetic networks operate in the cell. Early efforts included the production of simple networks capable of producing genetic oscillators and toggle switches. This week we will discuss two examples of artificial networks that led to many of the recent attempts at understanding network function and the consequences of gene expression variability. |
| 3 | Noise in gene expression (I) | {{< anchor "One_of_the_first_findings" >}}{{< /anchor >}}One of the first findings in systems biology was that gene expression is a fundamentally stochastic phenomenon. In particular, it appears that the randomness inherent to the biochemical processes involved in gene expression can lead to significant cell-to-cell variability in the numbers of proteins and mRNAs, leading to "non-genetic individuality" among genetically identical organisms. This week we discuss a classic study of variability in bacterial gene expression and introduce the concepts of extrinsic and intrinsic noise. We also will read a paper that discusses some of the theoretical underpinnings of stochastic chemical kinetics and present concepts that have been widely applied to stochastic gene expression. |
| 4 | Noise in gene expression (II) | {{< anchor "Lately_a_number" >}}{{< /anchor >}}Lately a number of studies have indicated that stochastic gene expression in eukaryotic cells can result in even greater cell-to-cell variability than that observed in bacteria. This variation is thought to happen because transcription in these organisms occurs in bursts rather than at a steady rate, leading to very broad population distributions of mRNAs and proteins. This week we will discuss two papers that examine stochastic gene expression in eukaryotes. One is a classic study of variability in cultured cells, and another is a study that utilizes quantitative PCR-based methods to count mRNAs in individual cells. |
| 5 | Noise in gene expression (III) |  {{< br >}}{{< br >}} {{< anchor "Now_that_stochastic" >}}{{< /anchor >}}Now that stochastic gene expression has been established as being an important biological effect, researchers are delving ever more deeply into the process of gene expression itself. This analysis includes trying to understand what biochemical reactions are the most prone to stochastic behavior and thus contribute the most to cellular variability. This week's papers include a theoretical examination of the sources of stochastic gene expression as well as a remarkable real-time visualization of stochastic gene expression. {{< br >}}{{< br >}} _Further Information on stochastic gene expression:_ {{< br >}}{{< br >}} \- MATLAB® code implementing Gillespie Algorithm ([M](/coursemedia/7-342-systems-and-synthetic-biology-how-the-cell-solves-problems-fall-2010/3c0f2e24cda2a4211398515b0c7696ef_MIT7_342_F10_algorithm.m))   {{< br >}}{{< br >}}   This is a Gillespie stochastic simulation of a constitutively expressed {{< br >}}{{< br >}}   gene. {{< br >}}{{< br >}} \- Stochastic gene expression dynamics ([PDF]({{< baseurl >}}/sections/lecture-summaries/mit7_342_f10_output.pdf)) {{< br >}}{{< br >}}    This is the output of the MATLAB code that implements the Gillespie  {{< br >}}   Algorithm for a constitutively expressed gene. {{< br >}}{{< br >}}  |
| 6 | Structure of biological networks | {{< anchor "One_of_the_central_goals" >}}{{< /anchor >}}One of the central goals of systems biology is to understand how the wiring of genetic networks allows them to perform cellular functions. While much is now known about what sets of genes interact, only recently have we begun to understand why those particular sets of interactions are useful. Through this week's papers, we will learn about efforts to find functional units ("network motifs") within large gene interaction networks and also check whether a particularly common motif actually work as theory predicts. |
| 7 | Physical limits of gradient and concentration sensing | {{< anchor "Microorganisms" >}}{{< /anchor >}}Microorganisms live in an environment very different from the world that we are used to. This week we will discuss a paper describing the basic physical challenges facing a cell. Although physics in the regime of small size and high viscosity make some things impossible, microbes have nevertheless found some surprising solutions to the challenges they face. |
| 8 | Field trip | {{< anchor "This_week" >}}{{< /anchor >}}This week we will attend the "Boston Area Systems and Synthetic (BASS) Biology Meeting" at the [Harvard-MIT Broad Institute](http://www.broadinstitute.org/). Hyun will be one of the three speakers at the seminar! The students will hear exciting current research in systems and synthetic biology taking place in the Boston area – the "silicon valley" for systems and synthetic biology. |
| 9 | Bacterial chemotaxis | {{< anchor "This_week_we_discuss" >}}{{< /anchor >}}This week we discuss two papers that probe the ability of _E. coli_ to swim towards food sources. For several decades it has been known that this chemotaxis occurs by alternating "runs," in which the cell swims straight, followed by "tumbles," in which the orientation of the cell is randomized. The cell is able to move towards food sources by altering the frequency of these tumbles depending upon whether things are improving or getting worse. This week's papers show that because the steady-state frequency of tumbling is independent of the concentration of food sources (robustness), the cell is able to respond to gradients over a wide range of concentrations. |
| 10 | Development of multicellular organisms under noisy conditions | {{< anchor "Last_week" >}}{{< /anchor >}}Last week, we discussed how the physical limits placed on how accurately a cell can measure gradients and concentrations can affect microorganisms. This week we discuss how multicellular organisms cope with the same physical limits. We will discuss two recent systems biology approach to studying cells in the _Xenopus_ (frog) and _Drosophila_ (fruit fly) embryos as examples. |
| 11 | Circadian oscillations | {{< anchor "Circadian_oscillations" >}}{{< /anchor >}}Circadian oscillations are 24-hour cellular oscillations that can be entrained by the day-night cycle—they are responsible for many phenomena, including, for instance, jet lag. This week, we discuss a paper that examines the circadian clock in cyanobacteria and another paper about the very different circadian clock used in _Drosophila_, which started off the vibrant study of circadian oscillations. |
| 12 | Noise in development | {{< anchor "The_developmental" >}}{{< /anchor >}}The developmental of an organism from a single cell to an adult involves one of the most complex biological programs in existence. Yet despite its complexity, the program usually executes with remarkable fidelity, a prime example of biological robustness. However, there are a few important instances where variability, arising from stochastic gene expression or otherwise, can lead the program to alternate endings. This week, we explore one paper in which stochastic gene expression is used by an organism to produce useful variability in development and another paper in which a different mechanism leads to randomly determined cell fates in _C. elegans_. |
| 13 | Synthetic biology | {{< anchor "At_the_beginning" >}}{{< /anchor >}}At the beginning of this course we read several papers in which very simple genetic networks were constructed. These projects allowed experimentalists to test models of gene expression and also experimentally provided a step towards engineering entirely new cellular functions. This week we will discuss two short papers describing the increasingly ambitious efforts of synthetic biologists to bend nature to man's will.